vnext/Shared/InspectorPackagerConnection.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <hermes/hermes.h>
+#include <hermes/DebuggerAPI.h>
+#include <hermes/inspector/Inspector.h>
+#include <hermes/inspector/chrome/Registration.h>
+#include <jsinspector/InspectorInterfaces.h>
+
+#include <WinRTWebSocketResource.h>
+
+
+namespace Microsoft::ReactNative {
+
+class InspectorPackagerConnection final {
+ public:
+  InspectorPackagerConnection(std::string&& url);
+  
+ private:
+  friend class RemoteConnection;
+  void sendMessageToPackager(std::string &&message) const;
+  void sendMessageToVM(int64_t pageId, std::string &&message);
+  
+  std::unordered_map<int64_t, std::unique_ptr<facebook::react::ILocalConnection>> m_localConnections;
```

Comment:
is there ever more than 1?

Reply:
In RNW apps it shouldn't be more than one .. But in Office, we allow multiple runtimes .. and hence local connections.